File: app/src/main/java/com/keplersegg/myself/Async/ISetUser.java
package com.keplersegg.myself.Async;

import com.keplersegg.myself.Helper.TokenType;
import com.keplersegg.myself.Interfaces.IErrorMessage;
import com.keplersegg.myself.Models.User;

public interface ISetUser extends IErrorMessage {

    void setUser(User user);

    void setToken(TokenType tokenType, String token);
}

File: app/src/main/java/com/keplersegg/myself/Interfaces/ISetUser.kt
package com.keplersegg.myself.Interfaces

import com.keplersegg.myself.Helper.TokenType
import com.keplersegg.myself.Models.User

interface ISetUser : IErrorMessage {

    fun setUser(user: User?, tokenType: TokenType)

    fun setToken(tokenType: TokenType, token: String?)
}
